backup/backup.go

```diff
@@ -314,7 +314,7 @@ func backupData(tables []Table) {
 		initialPipes := CreateInitialSegmentPipes(oidList, globalCluster, connectionPool, globalFPInfo)
 		// Do not pass through the --on-error-continue flag or the resizeClusterMap because neither apply to gpbackup
 		utils.StartGpbackupHelpers(globalCluster, globalFPInfo, "--backup-agent",
-			MustGetFlagString(options.PLUGIN_CONFIG), compressStr, false, false, &wasTerminated, initialPipes, true, false, 0, 0, gplog.GetVerbosity())
+			MustGetFlagString(options.PLUGIN_CONFIG), compressStr, false, false, wasTerminated.Load(), initialPipes, true, false, 0, 0, gplog.GetVerbosity())
```

Comment:
It seems to be not the best idea to pass a copy of an atomic value to the function. And semantically it differs from existing version. I suggest to pass a pointer to wasTerminated here, as before.

Reply:
done